src/main/java/Duke.java

```diff
@@ -111,6 +154,10 @@ public static void chat(String message) {
         System.out.println(UI_LINE_SPACING + message + UI_LINE_SPACING);
     }
 
+    public static void error(String message) {
+        System.out.println(UI_LINE_SPACING + ":( OOPS: " + message + "\n" + UI_LINE_SPACING);
+    }
+
     public static void end() {
         System.out.println(UI_LINE_SPACING + "Bye! Hope to see you again!" + UI_LINE_SPACING);
         isEnd = true;
```

Comment:
Overall code is clear and the logic flows and should have no problem completing the IP